extensions/amp-story/1.0/_locales/en.js

```diff
@@ -160,10 +160,15 @@ export default /** @const {!LocalizedStringBundleDef} */ ({
     description: 'Button label for the share target that shares a link via ' +
         'WhatsApp.',
   },
+  [LocalizedStringId.AMP_STORY_TOOLTIP_EXPAND_INSTAGRAM]: {
+    string: 'Expand Post',
+    description: 'Label in the tooltip text that appears when an Instagram ' +
+        'interactive embed is clicked. It invites users to expand the embed.',
```

Comment:
Now that we're going to use the internal translation process, we need to keep in mind that people who will translate this string will have zero external context. They might not even know what product it's for, or where it will be displayed. :(

Explaining that it's the label of a button that gets visible when X, and that it'll do Y when clicked, should be enough context (I imagine "expand" will be a hard word to translate when you can't see the animation, or don't know it's an animation)